typings/resin-sdk.d.ts

```diff
@@ -147,6 +147,40 @@ declare namespace ResinSdk {
 		choicesLabels?: { [key: string]: string };
 	}
 
+	interface BillingInformation {
+		id: number;
+	}
+
+	interface Organization {
+		id: number;
+		created_at: string;
+		name: string;
+
+		is_paid_for_via__billing_information: NavigationResource<
+			BillingInformation
+		>;
```

Comment:
This should definitely have changed in the API before orgs gets merged, as on master we now have proper billing info & plan details attached to the model.

Reply:
Indeed, had that removed locally but started rebasing on top of v10 and never pushed that.